src/slidem-deck.js

```diff
@@ -17,7 +17,8 @@ document.adoptedStyleSheets = [...document.adoptedStyleSheets, globalStyle];
  */
 const STATE_RE = /^#(?:slide-(?<slide>\d+))?(?:\/step-(?<step>\d+|Infinity))?/;
 
-export class SlidemDeck extends HTMLElement {
+// https://github.com/evanw/esbuild/issues/2800#issuecomment-1378198088
```

Comment:
I'm reviewing this issue and the codebase here, but it looks like it does not apply. We are not trying to access the class itself during a static code block, only in blocks that are not evaluated during parsing of the classes.

If I remove the IIFEs everything still works for me. Did you actually encounter any issues?

Reply:
When you ran the build, the output did not contain __privateGet?

Reply:
It does, but is that a problem? It works

Reply:
Oh I see now, it pushes a bunch of extra cruft to the output build.

Is that important? It's such a tiny size difference compared to the normal build, and it's caused by a "bug" in the build tooling. I'm not a fan of making suboptimal changes to our source to work around bugs in the build toolchain. If performance here was super important I'd consider making a change like that, but I find it hard to argue in favor of such a change for this codebase.